packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-utils.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { AUTH_STATUS_CODES, isNavigationFromDerivGO, isNavigationFromP2P } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import VerificationStatusActionButton from '../../../Components/verification-status-action-button';
+
+export const getFilesDescriptions = () => ({
```

Comment:
can we move this to constant file

Reply:
According to discussion, moved all these files to configs